crates/fuel-core/src/state/historical_rocksdb/modifications_history.rs

```diff
@@ -26,11 +38,17 @@ where
     type OwnedValue = Self::Value;
 }
 
-impl<Description> TableWithBlueprint for ModificationsHistory<Description>
+pub type ModificationsHistoryV1<Description> =
+    ModificationsHistoryVersion<Description, 0>;
+pub type ModificationsHistoryV2<Description> =
+    ModificationsHistoryVersion<Description, 1>;
+
+/// Blueprint for Modifications History V1. Keys are stored in little endian order
+/// using the `Column::HistoryColumn` column family.
+impl<Description> TableWithBlueprint for ModificationsHistoryV1<Description>
 where
     Description: DatabaseDescription,
 {
-    // TODO: The Blueprint should be `Plain<Primitive<8>, Postcard>` to sort
     //  the keys in the database. https://github.com/FuelLabs/fuel-core/issues/2095
```

Comment:
We might as well remove the second line of the comment.

Reply:
Done: d56f351f9ba666fadd507ce3839b79c766f9345e